src/certs/index.ts

```diff
@@ -4615,3 +4676,4 @@ L5/ndtFhKvshuzHQqp9HpLIiyhY6UFfEW0NnxWViA0kB60PZ2Pierc+xYw5F9KBa
 LJstxabArahH9CdMOA0uG0k7UvToiIMrVCjU8jVStDKDYmlkDJGcn5fqdBb9HxEG
 mpv0
 -----END CERTIFICATE-----
+`;
```

Comment:
I am still conscious about the size of the file. Is there a way to avoid it somehow for people who are sure that they're not needed?

Reply:
First of all I would suggest to check behaviour of webpack in current implementation. Since function which uses `certs` module is wrapped with `if (process.ENV...) {` (webpack is able to parse such), I guest webpack's tree-shaking may be able to cut `certs` module out since it not used.

Reply:
I am also concerned about the size of this file. It seems to me that it is more correct to do it as separate files in the original format (*.pem).

Reply:
@zeruk this will get us back to all these problems related to compiling bundlers like esbuild. I am supporting this pull request but hoping there is more optimal way to deal with the necessity of including such a large file.
We can also just agree here that technically the package does not change in size so we should just accept this PR and think of better solution in the future

Reply:
@esseswann we've discussed about it with @DavyJohnes and we came to the idea that it would be better to include the contents of `.pem` files inside `.json`. This will both _separate the code_ from the text(certs) and allow us to include these files _inside the bundles_. So I believe that's solution.